web/modules/custom/rdf_entity/rdf_entity.module

```diff
@@ -2,6 +2,152 @@
+function rdf_entity_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if ($form_id != 'field_storage_config_edit_form') {
+    return;
+  }
+  // Only add mapping form element to rdf entities fields.
+  if ($form_state->get('entity_type_id') != 'rdf_entity') {
+    return;
+  }
+  $form_obj = $form_state->getFormObject();
+  /** @var FieldStorageConfig $entity */
+  $entity = $form_obj->getEntity();
+  $schema = $entity->getSchema();
+  $form['rdf_mapping'] = array(
+    '#type' => 'details',
+    '#title' => t('Rdf field mapping'),
+    '#description' => t('This field uses a Sparql backend. Please map the fields to their corresponding rdf properties.'),
+    '#weight' => 99,
+  );
+  foreach ($schema['columns'] as $column => $column_desc) {
+    $description = $column_desc['description'] . "<br>";
+    foreach (array('type', 'length', 'size', 'serialize') as $key) {
+      if (!empty($column_desc[$key])) {
+        $description .= '<strong>' . $key . "</strong>: " . $column_desc[$key] . ' ';
+      }
+    }
+
+    $form['rdf_mapping'][$column] = array(
+      '#type' => 'textfield',
+      '#title' => $column,
+      '#description' => $description,
+      '#weight' => 150,
+      '#default_value' => $entity->getThirdPartySetting('rdf_entity', 'mapping_' . $column, FALSE),
+    );
+  }
+
+  $form['#entity_builders'][] = 'rdf_entity_form_alter_builder';
+}
+
+/**
+ * Entity builder callback: Save rdf field mapping.
+ */
+function rdf_entity_form_alter_builder($entity_type, FieldStorageConfig $entity, FieldStorageConfigEditForm &$form, FormStateInterface $form_state) {
+  $schema = $entity->getSchema();
+  foreach ($schema['columns'] as $column => $column_desc) {
+    $value = $form_state->getValue($column);
+    $entity->setThirdPartySetting('rdf_entity', 'mapping_' . $column, $value);
+  }
+}
+
+/**
+ * Implements hook_rdf_entity_view().
+ *
+ * @todo Remove this.
+ * Prints the bundle when viewing the rdf in full.
+ */
+function rdf_entity_rdf_entity_view($build, $entity, $display, $view_mode) {
+  if ($entity instanceof \Drupal\rdf_entity\Entity\Rdf && $view_mode == 'full') {
+    drupal_set_message(
+      t('Current page - Bundle: %bundle', array(
+        '%bundle' => $entity->bundle(),
+      )),
+    'info'
+    );
+  }
+}
+
+/**
+ * Implements hook_theme().
+ */
+function rdf_entity_theme() {
+  return array(
+    'rdf_entity' => array(
+      'render element' => 'elements',
+    ),
+  );
+}
+
+/**
+ * Prepares variables for rdf entity templates.
+ *
+ * Default template: rdf-entity.html.twig.
+ *
+ * Themes should use their own copy of rdf-entity.html.twig.
+ * The default is located  inside "templates/rdf-entity.html.twig".
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - elements: An array of elements to display in view mode.
+ *   - rdf_entity: The rdf_entity object.
+ *   - view_mode: View mode; e.g., 'full', 'teaser', etc.
+ */
+function template_preprocess_rdf_entity(&$variables) {
```

Comment:
This code is running, but I only see the title of the entity after I imported a federated repository in my triple store, and looked at it on the path `rdf_entity/http%3A%5C%5Cadministracionelectronica.gob.es%5Cctt`.

Am I missing something?